modules/ducktests/tests/ignitetest/services/utils/control_utility.py

```diff
@@ -105,18 +105,22 @@ def deactivate(self):
 
     @staticmethod
     def __parse_cluster_state(output):
-        state_pattern = re.compile("Cluster state: ([^\\s]+)")
-        topology_pattern = re.compile("Current topology version: (\\d+)")
-        baseline_pattern = re.compile("Consistent(Id|ID)=([^\\s]+),\\sS(tate|TATE)=([^\\s]+),?(\\sOrder=(\\d+))?")
+        state_pattern = re.compile("Cluster state: (?P<cluster_state>[^\\s]+)")
+        topology_pattern = re.compile("Current topology version: (?P<topology_version>\\d+)")
+        baseline_pattern = re.compile("Consistent(Id|ID)=(?P<consistent_id>[^\\s]+),\\sS(tate|TATE)=(?P<state>[^\\s]+),"
+                                      "?(\\sOrder=(?P<order>\\d+))?")
 
         match = state_pattern.search(output)
-        state = match.group(1) if match else None
+        state = match.group("cluster_state") if match else None
 
         match = topology_pattern.search(output)
-        topology = int(match.group(1)) if match else None
+        topology = int(match.group("topology_version")) if match else None
 
-        baseline = [BaselineNode(consistent_id=m[1], state=m[3], order=int(m[5]) if m[5] else None)
-                    for m in baseline_pattern.findall(output)]
+        baseline = []
+        for match in baseline_pattern.finditer(output):
```

Comment:
match variable redefinition makes code unreadable for me.

Reply:
There are several matches above. This is the same type Match, so what's the problem.

Reply:
you changing the type and content. 
brief reading may (will) confuse the reader.

Reply:
No, you are not right. Type is the same (see docs to finditer, it returns iterator of Match object)

Reply:
variables should be immutable.
you already have a 'match' variable, what's the reason to reassign it with other value? 
why not use a new name?

Reply:
LOLWUT? Reference should be immutable? Are we using Haskell or Scala? 
As for me, your proposition sounds like let's create mess with dozens variables, that do the same. 
I disagree, so I'd better close my PR, I'm tired of this.

Reply:
No chances %)